data/json/npcs/talk_tags.json

```diff
@@ -841,7 +1068,18 @@
       "Fight or flight?",
       "Weapons hot!",
       "Lock and load!",
-      "We've got incoming!"
+      "We've got incoming!",
+      "Yo, eyes open!",
+      "Alright, what the hell is going on?",
+      "Shit, <name_g>, we might have trouble here!",
+      "Things just went from bad to worse!",
+      "We're screwed man, screwed!",
+      "<swear!>, today was going so well too.",
+      "Looks like a storm's a'brewin!",
```

Comment:
`"Looks like a storm's a'brewin'!",`?

Reply:
An additional apostrophe at the end there makes gramatical sense, good spot!